libs/common-services/src/test/java/uk/gov/di/ipv/core/library/helpers/TestVc.java

```diff
@@ -71,7 +71,6 @@ public static class TestEvidence {
         private Integer validityScore;
         private Integer verificationScore;
         @Builder.Default private List<Object> ci = Collections.emptyList();
-        @Builder.Default private List<Object> ciReasons = Collections.emptyList();
```

Comment:
This doesn't appear in the schema - if it's present in real VCs then we may need to add it back